client/src/app/deck/deck-edit.tsx

```diff
@@ -486,7 +487,15 @@ function EditDeck({
 				onCancel={() => setShowUnsavedModal(!showUnsavedModal)}
 				onConfirm={back}
 			/>
-			<DeckLayout title={title} back={handleBack} returnText="Deck Editor">
+			<ConfirmModal
+				setOpen={showClearCardsModal}
+				title="Clear all cards"
+				description="Remove all cards?"
```

Comment:
```suggestion
				title="Remove all cards"
				description="Are you sure you want to remove all the cards in your deck?"
```
The current title is inconsistent with the button and the description is worded weirdly